seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -454,16 +453,14 @@
       "challengeSeed": [
         "let a = 8, b = 6;",
         "/* Alter code below this line */",
-        "a = b;",
-        "b = a;",
         "/* Alter code above this line */",
         "console.log(a); // should be 6",
         "console.log(b); // should be 8"
       ],
       "tests": [
-        "// Test a is 6",
-        "// Test b is 8",
-        "// Test destructuring was used"
+        "assert(a === 6, 'message: Value of <code>a</code> should be 6, after swapping.');",
+        "assert(b === 8, 'message: Value of <code>b</code> should be 8, after swapping.');",
+        "assert(/\\[\\s*.{1}\\s*,\\s*.{1}\\s*\\];/g.test(code), 'message: Use array destructuring to swap a and b.');"
```

Comment:
@Greenheart Did the changes. I was getting some merge conflicts, due to [this commit](https://github.com/freeCodeCamp/freeCodeCamp/commit/565825671504fff11622fe7190bc768982f9e074). The `challengeSeed` and `description` now has updated code.